frontend/SettingsPane.js

```diff
@@ -196,7 +196,11 @@ const InspectMenuButton = ({ isInspectEnabled, onClick, theme }) => (
 );
 
 const SettingsMenuButton = ({ onClick, theme }) => (
-  <button onClick={onClick} style={styles.settingsMenuButton}>
+  <button
+    onClick={onClick}
+    style={styles.settingsMenuButton}
+    title="Customize devtools"
```

Comment:
Maybe "Customize React DevTools". I don't think we use "devtools" capitalization anywhere except package name.